Sources/MapboxCoreNavigation/NativeHandlersFactory.swift

```diff
@@ -93,15 +93,7 @@ class NativeHandlersFactory {
     }()
     
     lazy var configHandle: ConfigHandle = {
-        let historyAutorecordingConfig = [
```

Comment:
Does auto-recording work by default now?

Reply:
It's disabled by default, you need to call `startRecording` explicitly now.